integrate-kobiton-travisci.md

```diff
@@ -0,0 +1,190 @@
+
+  Go to https://portal.kobiton.com and login to your Kobiton account.
+  * In the top navigation bar, select **"Devices"**.
+    ![](assets/1_device_bar.jpg)
+  * Hover over any device you want to test with and click on the Automation settings button (the gear symbol)   
+  
+    ![](assets/2_get_device.jpg)
+  * You can choose your preferred language, app type. In this guideline, we will choose **"NodeJS"** for *Language* and **"Hybrid/Native from Apps"** for *App type*. Copy the code on the right to prepare for the next step.
+  
+    ![](assets/2_kobiton_device.jpg)
+
+  **2.3 Configure automation test project**
+  
+  Kobiton already provides some samples for automation test, visit [here](https://github.com/kobiton/samples) for reference.
+
+  **In this guideline, we will use the `Node.js` sample (`samples/javascript` folder) as an example.**
+
+  Copy the sample to your GitHub repository and edit your `.travis.yml` file like below:
+  ```yml
+  language: node_js
+  node_js:
+    - '7'
+
+  script: <AUTOMATION_SCRIPT_RUN_COMMAND>
+  ```
+  > For example: using our provided sample, the automation script run command could be `npm run android-app-test` or `npm run ios-app-test`
+  
+  Open automation test script file in your repository or create a new one.
+
+  Remember to add `kobitonServerConfig` and `desiredCaps` in your script with ones you got from the previous step.
+
+  ```javascript
+  const username = '<YOUR_KOBITON_USERNAME>'
+  const apiKey = '<YOUR_KOBITON_API_KEY>'
+
+  var kobitonServerConfig = {
+    protocol: 'https',
+    host: 'api.kobiton.com',
+    auth: `${username}:${apiKey}`
+  }
+
+  var desiredCaps = {
+    sessionName:        'Automation test session',
+    sessionDescription: '', 
+    deviceOrientation:  'portrait',  
+    captureScreenshots: true, 
+    app:                '<APP_URL>', 
+    deviceGroup:        'KOBITON', 
+    deviceName:         '<KOBITON_DEVICE_NAME>',
+    platformVersion:    '<KOBITON_DEVICE_VERSION>',
+    platformName:       '<DEVICE_PLATFORM_NAME>' 
+  } 
+  ```
+
+>For more information on how to run automation test on Kobiton, visit:
+  >- Kobiton documentation: https://docs.kobiton.com/automation-testing/automation-testing-with-kobiton/
+  >- Kobiton blog: https://kobiton.com/blog/automation-web-appium-kobiton-nodejs/
+
+## 3. Run automation script on Kobiton devices
+ Push your changes to GitHub.
+
+ Travis CI will install the neccessary dependencies and then run the test on Kobiton. 
+  ![](assets/1_build_complete.jpg)
+
+  Go to https://portal.kobiton.con/sessions to check your testing session status.
+
+  ![](assets/2_kobiton_result.jpg)
+
+## 4. Fetch session data through REST API
+Kobiton already provides a Node.js sample on how to get session information using Kobiton REST API.
+
+Go to https://github.com/kobiton/samples/rest-api and follow the instruction.
```

Comment:
https://github.com/kobiton/samples/rest-api is not a valid link.

Reply:
This is true because the PR haven't merge yet